ACadSharp/IO/DXF/DxfStreamWriter/DxfSectionWriterBase.Entities.cs

```diff
@@ -16,13 +18,19 @@ protected void writeEntity<T>(T entity)
 				case Mesh:
 				case MLine:
 				case Solid3D:
-				case MultiLeader:
 				case Wipeout:
 					this.notify($"Entity type not implemented : {entity.GetType().FullName}", NotificationType.NotImplemented);
 					return;
 			}
 
-			this._writer.Write(DxfCode.Start, entity.ObjectName);
+			if (entity.ObjectName == "MLEADER")
+			{
+				this._writer.Write(DxfCode.Start, "MULTILEADER");
```

Comment:
The `ObjectName` is the equivalent of the DxfName for that object, I've noticed that MLEADER is not used either in a DXF file or a `DxfClass`, it would simplify things if we change the token in `DxfFileToken.EntityMLeader` for the valid name which is MULTILEADER not the one found in the documentation website.

This change will also allow you to remove the mapping dictionary in the DwgObjectWriter _classNameMap.

Reply:
done

Reply:
done